lib/CL/devices/tbb/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+#=============================================================================
+#   CMake build system files
+#
+#   Copyright (c) 2014 pocl developers
```

Comment:
maybe adapt year? if in doubt stay consistent with existing code

Reply:
I've seen different copyright claims in pocl source files: Sometimes "pocl developers", sometimes an individual person or institution. I'm wondering if the year should be updated if a file changes. Even if these files are added as new here, these files are copies of already existing files with some (minor) changes. Thus, I would apply the "How-to-adapt-the-copyright-if-a-file-is-changed"-policy of pocl and not the "Copyright-for-new-files-of-an-expternal-contributer"-policy in case such policies exist on the pocl side. Maybe I should ask the pocl developers about how they like to handle the copyright claims. I don't feel the necessaries to add my name/institution along the already existing names/institutions.

Reply:
maybe put a sentence on this in the final pull request text, there is a central authors file I think, so I assume pocl developers is more recent